### ISSUE TITLE
Linux build, no text at all

### ISSUE THREAD

**Issue by cubiq** [Ultimaker/cura-build#185]
I was able to build the latest master and the 3.4 branch under Ubuntu Xenial. The AppImage seems to be created successfully but when I run Cura I get absolutely no text. I bet it must be some very stupid thing missing but I can't find anything in the build logs. Any idea?

![no-text](https://user-images.githubusercontent.com/427614/44109907-8fc181b8-9ffe-11e8-806b-2d6def1a51eb.jpg)

The error log looks like this https://paste.debian.net/1037952/

**Comment by cubiq** [comment]
I solved by compiling in ubuntu 18.04 and upgrading python from 3.5.2 to 3.5.6.

**Comment by Ximidar** [comment]
On Centos7 I Installed these packages and my build started putting out cura with the text intact
```
yum update -y && yum install -y \
    devtoolset-3-gcc \
    devtoolset-3-gcc-c++ \
    devtoolset-3-gcc-gfortran \
    cmake3 \
    curl \
    git \
    make \
    mesa-libGL \
    mesa-libGL-devel \
    openssl-devel \
    tar \
    which \
    x11vnc \
    xorg-x11-server-Xvfb \
    tigervnc-server \
    xterm \
    xdotool \
    libX11-devel \
    xcb-util \
    xz \
    fontconfig-devel \
    fontconfig \
    zlib-devel \
    fuse \
    fuse-devel \
    glib-devel \
    glib2-devel \
    glibc-core \
    google-noto-fonts-common* \
    libpng12 \
    libpng12-devel \
    xcb-util-renderutil \
    xcb-util-renderutil-devel \
    libxcb-devel \
    xcb-util \
    xcb-util-devel \
    xcb-util-cursor \
    xcb-util-cursor-devel \
    xcb-util-image-devel \
    libXi \
    libXi-devel \
    libXrender \
    libXrender-devel \
    libxcb \
    xcb-util-image \
    libxkbcommon-x11 \
    libxkbcommon-x11-devel \
    xorg-x11-xkb-utils-devel \
    xorg-x11-xkb-utils \
    libXfixes-devel \
    libXfixes 
```
Specifically I think it was the fontconfig package that fixed it.
